src/components/nms/providers/amqp/downloads/apachenmsamqp-v182.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This is the wrong place to add such files now. In order to include the NMS-AMQP component in the news feed etc it was transitioned to the same collections-based release addition process use previously for 5.x and Artemis. See #58 and following changes for details.

You should be adding the file in src/_nms_amqp_releases/apachenmsamqp-v182.md, e.g basing it on the contents of [src/_nms_amqp_releases/apachenmsamqp-v181.md](https://github.com/apache/activemq-website/blob/d896350ae0930d47dae73edd4c241b2ee5173631/src/_nms_amqp_releases/apachenmsamqp-v181.md ) (which I just made various updates to in order to resolve some issues: it not using https download links, not using downloads.apache.org, not linking the sha checksums, removing the tag details as we arent meant to blatantly direct everyday users to the source repo containing unreleased bits, and it not including details on verifying the integrity of downloaded files).

Reply:
@gemmellr Should be fine now.